app/markets/ActivityTable.tsx

```diff
@@ -177,9 +179,165 @@ export const ActivityTable = ({ id }: { id: string }) => {
   );
 };
 
+interface TradeRowProps {
+  activity: MergedTradeTransaction;
+  aiAgentsList: AiAgentsList;
```

Comment:
Instead of passing down a list, why not pass a bool value to show if is an AI agent? Otherwise we are passing an entire list to every row

Reply:
Or a function to determine if it is

Reply:
bool works.